ote/src/cljs/taxiui/app/controller/loader.cljs

```diff
@@ -0,0 +1,27 @@
+(ns taxiui.app.controller.loader
+  (:require [taxiui.app.routes :as routes]
+            [tuck.core :as tuck]))
+
+(defn- bag-counter
+  "Function for counting hits in a bag (associative data structure counting \"hits\")."
+  [m op k]
+  (let [r (op (or (get m k) 0))]
```

Comment:
Where does `op` come from?

Reply:
Second function parameter, the actual calls pass the operation (`inc` or `dec` in this case) that gets applied on rows 19 and 27.